framework/framework.go

```diff
@@ -160,38 +240,38 @@ func (f *Framework) ProcessDelete(obj interface{}, resources []Resource) error {
 //         UpdateFunc:    updateFunc,
 //     }
 //
-func (f *Framework) ProcessUpdate(newObj interface{}, resources []Resource) error {
+func ProcessUpdate(obj interface{}, resources []Resource) error {
```

Comment:
Just renaming this because there is no reason to differentiate. It is now the same as in `ProcessCreate`.